pages/popup/index.html

```diff
@@ -108,7 +108,7 @@ <h1>Indie Wiki Buddy</h1>
             <input id="crossLanguageCheckbox" type="checkbox" />
             <span id="crossLanguageIcon" aria-hidden="true"></span>
             Redirect non-English Fandom wikis to
-            indie English wikis there is no same-lang wiki
+            indie English wikis when there is no same-lang wiki
```

Comment:
You can actually undo this change - we're going to merge in the internationalization PR soon, which will conflict with this change (and will include the fix).